source/config-file/protect-connection-strings.txt

```diff
@@ -0,0 +1,71 @@
+.. _compass-protect-connection-strings:
+.. _hide-connection-string-credentials:
+
+==========================================
+Hide Credentials in Your Connection String 
+==========================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+You can use the ``protectConnectionStrings`` option to hide credentials when 
+users connect to |compass-short| with a connection string. When 
+``protectConnectionStrings`` is enabled, users cannot edit hidden credentials 
+in the |compass-short| connection form, copy the connection string in the 
+|compass-short| interface, or see the credentials when using the :ref:`Export 
+to Language <compass-export-query>` functionality. 
+
+Procedure
+---------
+
+To protect your connection string credentials, enable the 
+``protectConnectionStrings`` option. You can set this option in the 
```

Comment:
Consider making these options a bulleted list. 

You can set this option in either:

- x 
- y 
- z